src/main/java/edu/ucar/unidata/rosetta/repository/JdbcPropertiesDao.java

```diff
@@ -0,0 +1,66 @@
+package edu.ucar.unidata.rosetta.repository;
+
+import edu.ucar.unidata.rosetta.domain.RosettaProperties;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import org.springframework.dao.DataRetrievalFailureException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.jdbc.core.support.JdbcDaoSupport;
+
+/**
+ * @author oxelson@ucar.edu
+ */
+public class JdbcPropertiesDao extends JdbcDaoSupport implements PropertiesDao {
+
+    protected static Logger logger = Logger.getLogger(JdbcPropertiesDao.class);
+
+    private SimpleJdbcInsert insertActor;
+
+    /**
+     * Looks up and retrieves the persisted uploads directory.
+     *
+     * @return The persisted uploads directory.
+     * @throws DataRetrievalFailureException  If unable to lookup uploads directory.
+     */
+    @Override
+    public String lookupUploadDirectory() throws DataRetrievalFailureException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.springframework.dao.DataRetrievalFailureException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)